client/src/app/site/motions/components/motion-detail/motion-detail.component.html

```diff
@@ -360,6 +360,20 @@ <h4 translate>Origin</h4>
     >
         <!-- Line Number and Diff buttons -->
         <div *ngIf="!editMotion && !motion.isStatuteAmendment()" class="motion-text-controls">
+            <mat-form-field class="motion-goto-line" *ngIf="highlightedLineOpened">
+                <input type="number" [(ngModel)]="highlightedLineTyping" [ngModelOptions]="{standalone: true}" matInput
```

Comment:
![localhost_4200_motions](https://user-images.githubusercontent.com/10233032/51897987-3ad89700-23b0-11e9-86a6-e960ceded17e.png)
the input field it too close to the motion text.
Opening the input field during mobile mode puts unwanted line breaks into the text.